tests/org.eclipse.n4js.xpect.tests/xpect-tests/typesystem/polyExpressions/ArrLit_simple.n4js.xt

```diff
@@ -93,8 +93,8 @@ var v32b : Array<B> = [ b, ...myIterableOfC ];
 
 // XPECT type of '[ b, ...getArrOfC() ]' --> Array<A>
 var v33a : Array<A> = [ b, ...getArrOfC() ];
-// XPECT type of '[ b, ...myIterableOfC ]' --> Array<A>
-var v33b : Array<A> = [ b, ...myIterableOfC ];
+// XPECT type of '[ b, ...myIterableOfC ]' --> Array<B>
+var v33b : Array<B> = [ b, ...myIterableOfC ];
```

Comment:
Why did you replace `Array<A>` by `Array<B>`, here? Test cases v32b and v33b are now identical.

Reply:
Thanks for this hint! I fixed it!